README.md

```diff
@@ -278,9 +278,11 @@ feature is currently disabled by default. To enable this feature add the followi
 ```yaml
 external-sources:
   enable: true
+  abort-after: 10m
   maven:
-    search-upstream-by-sha1: true
```

Comment:
👋 thanks for adding the new field! Would it be possible to keep the field name here so that we don't release a breaking change? I know we're pre v1.0, but the best way to handle this is to deprecate this field and add the new one (allowing people to migrate) rather than changing it on the fly =)

Reply:
Oh wow - on second pass it looks like the json/yaml tags here were never correct here so thanks for the catch and docs update. I've made a note to change the json tag to be consistent going forward.